cmd/revad/svcs/httpsvcs/datasvc/get.go

```diff
@@ -30,7 +30,12 @@ import (
 func (s *svc) doGet(w http.ResponseWriter, r *http.Request) {
 	ctx := r.Context()
 	log := appctx.GetLogger(ctx)
-	fn := r.URL.Path
+	files, ok := r.URL.Query()["filename"]
```

Comment:
This should be optional, if the query parameters is present we use it, else we rely on the url path, but we don't fail with bad request, please change accordingly